.travis.yml

```diff
@@ -6,4 +6,36 @@ python:
   - 3.4
```

Comment:
@kennethreitz It seems Travis isn't running the tests any longer. Can you investigate that?

Reply:
flipped the switch